app/web-app/src/components/proposal-form.tsx

```diff
@@ -35,6 +35,7 @@ import { ScrollArea } from '@/components/ui/scroll-area';
 import ChevronLeft from 'lucide-react/dist/esm/icons/chevron-left';
 import { Button } from '@/components/ui/button';
 import ChevronRight from 'lucide-react/dist/esm/icons/chevron-right';
+import MoreInfo from './more-info';
```

Comment:
It's better to use absolute imports with path aliases (e.g., @/asdasd) for improved readability and maintainability instead of long relative paths like ../../asdasd.